core/src/main/resources/hudson/diagnosis/Messages.properties

```diff
@@ -6,4 +6,5 @@ HudsonHomeDiskUsageMonitor.DisplayName=Disk Usage Monitor
 
 NullIdDescriptorMonitor.DisplayName=Missing Descriptor ID
 ReverseProxySetupMonitor.DisplayName=Reverse Proxy Setup
+ReverseProxySetupMonitor.EncodedForwardSlashesDisallowed=Detected that encoded slashes are not allowed.  If you are running Tomcat, please be sure to run it with -Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true
```

Comment:
According to [this](https://stackoverflow.com/a/9933890/12916), Apache HTTPD also suffers from the same issue. Better to link to the [Jenkins wiki](https://wiki.jenkins.io/display/JENKINS/Jenkins+says+my+reverse+proxy+setup+is+broken) where the problem is described and solutions offered for various containers—as `doAct` already does.